hilti/runtime/src/tests/intrusive-ptr.cc

```diff
@@ -35,4 +37,243 @@ TEST_CASE("managed objects") {
     CHECK_EQ(Managed::instances, 0);
 }
 
+struct TestObject : intrusive_ptr::ManagedObject {
+    static uint64_t references;
```

Comment:
`s/references/instances/` ? I got confused with terminology for a second when looking at the tests (e.g., `TestObject::references == 1` when there are two intrusive ptrs)